modules/standard/CTypes.chpl

```diff
@@ -1161,7 +1166,8 @@ module CTypes {
     Returns a :type:`c_ptr` to the address of any chapel object.
 
     Note that the behavior of this procedure is identical to :func:`c_ptrTo`
-    for scalar types. It only differs for arrays, strings, and bytes.
+    for scalar types. It only differs for arrays, strings, bytes, and class
```

Comment:
How about "for scalar types and records" for the "identical" sentence?  I don't know whether we consider records scalar types, but I'd be surprised if we did (and think it's clearer mentioning them directly in any case).  We could potentially say "value types" except that in many contexts, I think we'd consider strings and bytes to be value types.

Reply:
Yeah I like "for scalar types and records", and I don't know if there's another good term to group the types we have special behavior for. I agree about "value types".